src/solver/optimisation/opt_construction_matrice_des_contraintes_outils.cpp

```diff
@@ -30,8 +30,8 @@ void OPT_ChargerLaContrainteDansLaMatriceDesContraintes(
   int NombreDeTermesDeLaContrainte,
   char SensContrainte)
 {
-    int& nombreDeTermes = ProblemeAResoudre->NombreDeTermesDansLaMatriceDesContraintes;
```

Comment:
Does not using a reference here make any difference ?

Reply:
forget it! this file is deleted in develop